packages/angular/cli/models/parser.ts

```diff
@@ -93,21 +93,28 @@ function _coerce(str: string | undefined, o: Option | null, v?: Value): Value |
 
 
 function _getOptionFromName(name: string, options: Option[]): Option | undefined {
-  const cName = strings.camelize(name);
+  const camelName = /(-|_)/.test(name)
+    ? strings.camelize(name)
+    : name;
 
   for (const option of options) {
-    if (option.name == name || option.name == cName) {
+    if (option.name === name || option.name === camelName) {
       return option;
     }
 
-    if (option.aliases.some(x => x == name || x == cName)) {
+    if (option.aliases.some(x => x === name || x === camelName)) {
       return option;
     }
   }
 
   return undefined;
 }
 
+function removeLeadingDashes(key: string): string {
```

Comment:
non-exported functions should start with a `_`.

Reply:
Updated.